test/unit/db/seeds_test.rb

```diff
@@ -80,7 +80,8 @@ def teardown
     assert_equal master_service.id, master_app_plan.issuer_id
     assert_equal 'enterprise', master_app_plan.name
 
-    assert_equal ApplicationPlan.find_by(name: 'Master Plan').id, master_account.bought_cinstance&.plan.id
+    assert(master_bought_cinstance = master_account.bought_cinstance)
```

Comment:
Apparently we don't need `master_bought_cinstance` again in the same test. Why setting a variable now?

Reply:
@guicassolato it is just a possible way to solve what @hallelujah said in https://github.com/3scale/porta/pull/1292#discussion_r330738420 of the nil check in tests. But I am opened to other proposals :smile: :+1:

Reply:
I'm all right with fixing the nil check. I'm just saying we don't need to set the variable.